### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Atharva0045**
Partially solves #1 
Couldn't get the hand of the contributing steps.

### CODE REVIEW

**Comment by Atharva0045** [comment]
@climbai can you share the contributing guide, can't get a hang of it?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,66 @@
...
+
+1. Clone the repository to stay updated:
+    ```bash
+    git clone https://github.com/yourusername/platform-name.git
+    cd platform-name
+    ```
+
+2. Join our efforts! If you're interested in contributing, check out the **[Contributing](#contributing)** section below.
+
+
```
Comment (climbai) [inline README.md]: Add the contributing steps here!

**Review by climbai — CHANGES_REQUESTED** [review]
Add the contribution steps at the mentioned place..

**Comment by climbai** [comment]
Sure @Atharva0045, 
Here are the steps for contributing to the project.

## Contributing

We welcome early contributions as we develop the platform! Feel free to open issues, suggest features, or submit pull requests. Please follow these steps to contribute:

### Step-by-Step Contribution Workflow:

1. **Fork the Repository**: 
   - Fork the repository (this will create a copy of the repo under your GitHub account).
   
2. **Clone the Repository Locally**:
   - Clone your forked repository:
     ```bash
     git clone https://github.com/climbai/user_platform.git
     cd user_platform
     ```

3. **Create a Feature Branch**:
   - Create a new branch for your feature or bug fix:
     ```bash
     git checkout -b feature-name
     ```

4. **Push Changes to the Original Repo's `dev` Branch**:
   - Add the original repository as an upstream remote:
     ```bash
     git remote add upstream https://github.com/climbai/user_platform.git
     git fetch upstream
     ```

   - Push your changes to your local branch of the forked repository:
     ```bash
     git push origin feature-name
     ```

5. **Create a Pull Request**:
   - Open a pull request (PR) to merge your changes into the `dev` branch.
   - Make sure your PR includes a clear description of the changes and any relevant details.

6. **Review and Merge**:
   - After your PR is reviewed and validated, it will be merged into the `dev` branch.

**Comment by Atharva0045** [comment]
@climbai Thank you for the instant support.

**Review by climbai — APPROVED** [review]
Looks good to me!

**Pull request merged**